details.js

```diff
@@ -142,6 +142,6 @@ function closePopUp(closeId) {
     let closePop = document.querySelector('#_overlay');
     closePop.remove();
 }
-    /* eslint-enable */
+    /* es-lint enable */
```

Comment:
- [OPTIONAL] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉